package.json

```diff
@@ -6,33 +6,48 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "deploy:s3": "serverless client deploy --no-config-change --no-policy-change --no-cors-change",
```

Comment:
duplicates 'client:deploy' script

Reply:
@NikaOrl I don't see any duplication. Could you please check again? :)

Reply:
@Ilya-Valasiuk `serverless client deploy --no-config-change --no-policy-change --no-cors-change` and `sls client deploy --no-config-change --no-policy-change --no-cors-change` are the same scripts

Reply:
It's not an error, just a recommendation so the score is still 5/5 from my point of view